### PR TITLE
DDF-3719 Improves UI developer documentation

### PR DESCRIPTION
**Pull request by djblue**
#### What does this PR do?

- Updates `ui/README.md` with more getting started information
- Adds new `ace disable-idp` command

#### Who is reviewing it? 

@mackncheesiest 
@tbatie 

#### Select relevant component teams: 

@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.

@brendan-hofmann
@bdeining  

#### How should this be tested?

- Review documentation changes
- Quick build
- Start DDF
- Run `yarn disable-idp`
- Run `yarn start` for any ui
- Ensure ui works without any idp issues

#### What are the relevant tickets?

[DDF-3719](https://codice.atlassian.net/browse/DDF-3719)

#### Checklist:

- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process

Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/README.md [hunk 1]
```diff
@@ -5,20 +5,72 @@ for javascript projects in DDF.
 
 To skip this module in a maven build, specify the `-pl !\ui` build flag.
 
+**Warning:** If you choose to skip the ui module and the distribution
+module is unable to find a previously build artifact, the build will fail.
```
Comment (brendan-hofmann) [inline ui/README.md]: ✏️ previously *built*

File: ui/README.md [hunk 2]
```diff
@@ -5,20 +5,72 @@ for javascript projects in DDF.
...
+
 ## Getting Started
 
 All of the javascript is structured as npm packages located in the
 `packages` directory. They can have their own dependencies specified in
-the `package.json` file. Those dependencies can be from npm or packages in
-the `packages` directory. Package specific help may be provided in the
-package's `README.md`.
+the `package.json` file. Those dependencies can be from a npm registry or
```
Comment (brendan-hofmann) [inline ui/README.md]: ✏️ from *an* ?

File: ui/README.md [hunk 3]
```diff
@@ -5,20 +5,72 @@ for javascript projects in DDF.
...
+### Web Context Policy Manager
+
+The first issue most encounter when running the dev server is a redirect
+to the IDP server. Currently authentication through IDP does not work
+behind a proxy. Ideally, the IDP would work transparently behind a proxy,
+at which point this solution will no longer be necessary. Until then, to
+disable IDP we need to re-configure the web context policy manager.
+Luckily, it's as easy as:
+
+    yarn disable-idp
```
Comment (brendan-hofmann) [inline ui/README.md]: Yes. 😻

**Comment by djblue** [comment]
build now

File: ui/README.md [hunk 4]
```diff
@@ -5,20 +5,72 @@ for javascript projects in DDF.
...
+## Ace
 
 An existing example of a locally shared dependency is `ace`. It is a node
 cli for interoperating with the maven eco-system and is depended upon by
-any package that wishes to produce a web app. If you wish for `ace` to be
-accessible globally, do:
+any package that wishes to produce a web app. **Most of the time you won't
+need to use `ace` directly, but you may be using it indirectly through
+scripts specified in the `package.json`**. This information is mostly for
+those who need to know a little bit more about the ui build.
```
Comment (Lambeaux) [inline ui/README.md]: 💯

**Comment by djblue** [comment]
build now

**Comment by djblue** [comment]
build now

**Pull request merged**